src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles !== '')
+    .reduce((cssStyles, styles) => {
+      const [key, value] = styles.split(':').map((item) => item.trim());
+
+      if (key && value) {
+        cssStyles[key] = value;
+      }
+
+      return cssStyles;
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Align the chained methods properly.